examples/sdk-app-example/app/build.gradle.kts

```
@@ -0,0 +1,84 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.bugsnag.gradle)
+}
+
+android {
+    namespace = "com.example.bugsnag.android"
+    compileSdk = 35
+
+    defaultConfig {
+        applicationId = "com.example.bugsnag.android"
+        minSdk = 21
+        targetSdk = 35
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    signingConfigs {
+        create("config") {
+            keyAlias = "password"
+            keyPassword = "password"
+            storeFile = file("../fakekeys.jks")
+            storePassword = "password"
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
```

minification needs to be enabled on release builds:
```suggestion
            isMinifyEnabled = true
```